README.md

```diff
@@ -467,34 +470,46 @@ notification intent that the Klaviyo SDK requires in order to track opens when y
 
 ## In App Forms
 
+In-app forms are messages displayed to mobile app users while they are actively using your app. 
+You can create new in-app forms in a drag-and-drop editor in the Sign-Up Forms tab in Klaviyo.
+
+[//]: # (TODO -- add help doc link)
+
 ### Prerequisites
-- Using version 3.2.0 or higher
```

Comment:
Maybe overthinking it, but should we be explicit that this is referring to SDK version?